var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -48,6 +48,13 @@ def install(self, spec, prefix):
     @when('@1.5.90:')
     def install(self, spec, prefix):
         cmake_args = ['-GUnix Makefiles']
+
+        # For Fujitsu compiler(fj)
+        # Use spack compiler flags for cmake.
+        if self.compiler.name == 'fj':
+            cmake_args.extend(['-DCMAKE_C_FLAGS={0}'
+                              .format(env["SPACK_CFLAGS"])])
```

Comment:
@tkameyama can you set this as 

```
'-DCMAKE_C_FLAGS={0}'
.format(spec.compiler_flags['cflags'])
```

This is a slightly-more robust way to access the flags. Ideally we could use some of the built-in flag-handling logic for `CMakePackage` but it appears that this uses different build systems depending on the version.

Reply:
Thanks for your advice!
I changed way to take compiler flags on commit of `libjpeg-turbo: fix way to take compiler flags`. 

`spec.compiler_flags['cflags']` was list, so I used phrase `.join`.